### PR TITLE
Adding the remaining topologies of cardinality 3

### PR DESCRIPTION
**Pull request by JoeF131**
As discussed in [#898](https://github.com/pi-base/data/pull/898#issuecomment-2469334620), there are only two missing spaces of cardinality 3 and as such it would be reasonable to add those.
Below is a list with all 9 topological spaces of cardinality 3 with the enumeration corresponding to the enumeration of the list on [Wikipedia](https://en.wikipedia.org/wiki/Finite_topological_space#3_points):
1. [S190](https://topology.pi-base.org/spaces/S000190)
2. new space P203
3. new space P204
4. [S164](https://topology.pi-base.org/spaces/S000164)
5. [S187](https://topology.pi-base.org/spaces/S000187)
6. [S7](https://topology.pi-base.org/spaces/S000007)
7. [S11](https://topology.pi-base.org/spaces/S000011)
8. [S188](https://topology.pi-base.org/spaces/S000188)
9. [S189](https://topology.pi-base.org/spaces/S000189)

### CODE REVIEW

File: spaces/S000203/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+uid: S000203
+name: 3 points with a basis consisting of a singleton
```
Comment (StevenClontz) [inline spaces/S000203/README.md:3]:
```suggestion
name: Three-point set with the basis $\{\{0\},X\}$
```

Follows the pattern of other spaces named as "three-point set"s. And avoids the subtlety of the definition of basis for when the whole space is the only neighborhood of a point.

Comment (JoeF131) [reply spaces/S000203/README.md:3]: Makes sense. I will change the name of P204 as well.

File: spaces/S000203/properties/P000010.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000203
+property: P000010
+value: false
+---
+
+The only basis of this space is $\{\{0\}\}$ and the closure of $\{0\}$ all of $X$ so $X$ is not semiregular.
```
Comment (StevenClontz) [inline spaces/S000203/properties/P000010.md:7]:
```suggestion
The set $\{0\}$ belongs to every basis generating the space, but $\mathrm{int}(\mathrm{cl}(\{0\}))=X$.
```

**Review by StevenClontz — APPROVED** [review]
Thanks for accepting my suggestions. I think this is good! I like that every property (besides semimetrizable and Ascoli #861) is covered by this PR for the spaces.

**Comment by StevenClontz** [comment]
> > Thanks for accepting my suggestions. I think this is good! I like that every property (besides semimetrizable and Ascoli #861) is covered by this PR for the spaces.
> 
> My proof was simply wrong since X is also part of the basis which I missed.

If we're lazy, one might define a basis to just satisfy the second property B2 at https://en.wikipedia.org/wiki/Base_(topology)#:~:text=has%20countable%20weight.-,If,of%20a%20space,-%F0%9D%91%8B - then your basis is a basis (and to generate the topology, just throw in the entire space as well as all unions of basic open sets).

**Pull request merged**